services/web/server/src/simcore_service_webserver/catalog/_handlers.py

```diff
@@ -82,7 +83,9 @@ class ListServiceParams(PageQueryParameters):
 @_handlers_errors.reraise_catalog_exceptions_as_http_errors
 async def dev_list_services_latest(request: Request):
     request_ctx = CatalogRequestContext.create(request)
-    query_params = parse_request_query_parameters_as(ListServiceParams, request)
+    query_params: ListServiceParams = parse_request_query_parameters_as(
```

Comment:
I wonder if we have the same problem of decorators here ... 
Have a look in https://github.com/itisfoundation/osparc-simcore/blob/b77814ac1ded8f8ad89ff72c43fc4434e5612a5e/packages/service-library/src/servicelib/aiohttp/requests_validation.py

Reply:
will check

Reply:
hmm, that one is tricky